pkg/combustion/registry.go

```diff
@@ -61,11 +61,17 @@ func configureRegistry(ctx *image.Context) ([]string, error) {
 		return nil, fmt.Errorf("configuring helm: %w", err)
 	}
 
-	if err = configureEmbeddedArtifactRegistry(ctx, helmTemplatePath, helmManifestHolderDir); err != nil {
+	var configured bool
+	if configured, err = configureEmbeddedArtifactRegistry(ctx, helmTemplatePath, helmManifestHolderDir); err != nil {
```

Comment:
It's better to drop the inline if statement and use short form declaration `configured, err := ...`